README.md

```diff
@@ -1,24 +1,152 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
 
-Things you may want to cover:
+<!-- TABLE OF CONTENTS -->
 
-* Ruby version
+# :green_book: Table of Contents
+- [:green_book:Table of Contents](#-table-of-contents)
+- [:book: Recipe App](#Recipe-app)
+  - [:hammer_and_wrench: Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [:link: Project Link ](#-project-link-)
+  - [:computer: Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [:busts_in_silhouette: Authors ](#-authors-)
+  - [:telescope: Future Features ](#-future-features-)
+  - [:handshake: Contributing ](#-contributing-)
+  - [:star:️ Show your support ](#️-show-your-support-)
+  - [:pray: Acknowledgments ](#-acknowledgments-)
+  - [:memo: License ](#-license-)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+#  Budget-app <a name="about-project"></a>
+> This app focuses on planning a budget for any category a user chooses. it could be music, sports, entertainment etc. It helps plan how much is needed to fulfil a category. 
+## :hammer_and_wrench: Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
 
-* Database creation
+<!-- Features -->
 
-* Database initialization
+### Key Features <a name="key-features"></a>
+- We can add, remove, update and display recipe.
+- It has user authentication.
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* Deployment instructions
+<!-- GETTING STARTED -->
 
-* ...
+## :computer: Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+```
+ - Laptop/Desktop.
+ - Internet.
+ - Ruby.
+ - Rails.
+ - Vs Code.
+```
+### Setup
+Clone this repository to your desired folder:
+```
+ - mkdir my-folder
+ - cd my-folder
+ -https://github.com/Ade179/Budget-App.git
+ - cd Recipe-app
```

Comment:
- [ ] You are saying this `cd Recipe-app` which is not correct, would you add a correct command which should be `cd Budget-App`

Reply:
Done. Thank you